build/.vsts-ci.yml

```diff
@@ -447,6 +450,55 @@ jobs:
   - script: sudo rm -rf $(Agent.BuildDirectory)/*
     displayName: 'Cleanup'
     condition: always()
+- job: ubuntu2204build
+  container: linux-c-ubuntu-2204
+  pool:
+    name: 'sdk-c--ubuntu-18'
+  displayName: 'Ubuntu 22.04 Unit Tests (Debug)'
```

Comment:
```suggestion
  displayName: 'Ubuntu 22.04 (Debug)'
```

Reply:
All of these changes are from the merged changes in main. Do we want to have a difference of files/changes?

Reply:
I advise against making any changes here - this should be just a merge.
We can make the suggested changes in `main` then backport to LTS at al later time.